src/js/binaryen.js-pre.js

```diff
@@ -1 +1,2 @@
-var Binaryen = function(Module) {
```

Comment:
Changed this initially just to put everything into a closure. Now that a.js is instrumented instead of the production build and the Binaryen global exposed conditionally, it's actually necessary.